Bio/CAPS/__init__.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, **kwds):
 
 
 class AlignmentHasDifferentLengthsError(Exception):
+    """Manage exceptions for Alignments with different length."""
```

Comment:
Why "Manage"? Also having looked at the context it is raised, it would be more accurate to say something like "Exception where sequences in alignment have different lengths".

Looking at this, it might be better for this to be a subclass of ``ValueError`` - but that would probably be best as a separate pull request later.

Reply:
I had some issues with flake8 in the past D401, I would thought the suggested statement wouldn't pass; but it works well; the same for the below DocStrings, Maybe that doesn't affect Classes rather Methods.

Reply:
"Use imperative mood..." only applies to methods and functions, not to module and class docstrings.

Reply:
The idea here is that a class is an object (passive), but functions and methods are called to do something (active). Thus the "imperative mood" only applies to the descriptions of these actions.